zdict/dictionaries/yandex.py

```diff
@@ -57,8 +67,14 @@ def query(self, word: str):
 
         content_json = json.loads(content)
 
-        if content_json.get('code') != 200:
-            # TODO: handle response codes for different situation
+        status = content_json.get('code')
+        if status != 200:
+            # https://tech.yandex.com/translate/doc/dg/reference/translate-docpage/#codes
+            message = self.status_code.get(
+                status,
+                'Some bad thing happened with Yandex'
+            )
+            print('Yandex: ' + message)
```

Comment:
` print('Yandex:', message)`?
print will append a space for you.

Reply:
I like the concat by `+`, but if you like that one I can change it.